tooling/bin/generate-final-artifact.sh

```diff
@@ -152,6 +152,26 @@ for architecture in "${architectures[@]}"; do
             "${tmp_folder_final_musl_appsec}/etc/recommended.json"
     fi
 
+    ########################
+    # PHP Stubs
+    ########################
+    stubs=(
+        "$3/src/ddtrace_php_api.stubs.php"
+        "$3/ext/ddtrace.stub.php"
+        "$3/ext/hook/uhook.stub.php"
+        "$3/ext/hook/uhook_attributes.stub.php"
+    )
+
+    mergedStubs=""
+    for stub in "${stubs[@]}"; do
+        content=$(<"$stub")
+        content="${content#<?php}"
+        mergedStubs+="$content"
+    done
+
+    stub="<?php\n$mergedStubs"
+    echo -e "$stub" > "$packages_build_dir/datadog-tracer.stubs.php"
```

Comment:
I think it's this echo `-e` which causes that.
```suggestion
    stub=$'<?php\n'"$mergedStubs"
    echo "$stub" > "$packages_build_dir/datadog-tracer.stubs.php"
```

Not sure, but does this work?